lib/auth/machineid/machineidv1/workload_identity_service.go

```diff
@@ -313,9 +313,6 @@ func (wis *WorkloadIdentityService) SignX509SVIDs(ctx context.Context, req *pb.S
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	if err := authCtx.AuthorizeAdminAction(); err != nil {
```

Comment:
Hah - fair enough - I only added this endpoint fairly recently (last week) and I hadn't really made my mind up on whether it should be considered admin. When I added this functionality to `tsh` - I realised we don't generally consider these cert generation endpoints as admin actions and it prevents the headless functionality of `tsh` working correctly. So it pushed me from "Hmm, I should make this admin just in case" to "Hmm, admin isn't really necessary for this RPC"